project-2/src/components/layout/footer/footer-list/index.jsx

```diff
@@ -0,0 +1,53 @@
+import { List } from "antd";
+import "./style.scss";
+
+function FooterList(props) {
```

Comment:
```suggestion
function FooterList({ data: { title, list, ... } }) {
```
Có thể sử dụng destructuring để thay thế props.data.title, props.data.list,...